PdfScribe/Properties/Resources.resx

```diff
@@ -121,4 +121,73 @@
     <value>..\Resources\AdobeIcon.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="ErrorDialogInstructionCouldNotWrite" xml:space="preserve">
+    <value>Could not create the output file.</value>
+  </data>
+  <data name="ErrorDialogInstructionPDFGeneration" xml:space="preserve">
+    <value>There was a PDF generation error.</value>
+  </data>
+  <data name="ErrorDialogInstructionUnexpectedError" xml:space="preserve">
+    <value>There was an internal error. Enable tracing for details.</value>
+  </data>
+  <data name="ErrorDialogOutputFileAccessDenied" xml:space="preserve">
+    <value>Access denied - check permissions on output folder.</value>
+  </data>
+  <data name="ErrorDialogOutputFilenameInvalid" xml:space="preserve">
+    <value>Output file path is not valid. Check the "OutputFile" setting in the config file.</value>
+  </data>
+  <data name="ErrorDialogOutputFilenameTooLong" xml:space="preserve">
+    <value>Output file path too long. Check the "OutputFile" setting in the config file.</value>
+  </data>
+  <data name="ErrorDialogSpoolService" xml:space="preserve">
+    <value>This program is meant to be called by the spooler service</value>
+  </data>
+  <data name="ErrorDialogTextFileInUse" xml:space="preserve">
+    <value>{0} is being used by another process.</value>
+  </data>
+  <data name="ErrorDialogTextGhostScriptConversion" xml:space="preserve">
+    <value>Ghostscript error code {0}.</value>
+  </data>
+  <data name="ErrorExceptionMsg" xml:space="preserve">
+    <value>Exception message: {0}</value>
+  </data>
+  <data name="ErrorInsufficientPrivileges" xml:space="preserve">
+    <value>Insufficient privileges to either create or delete {0}</value>
+  </data>
+  <data name="ErrorInUse" xml:space="preserve">
+    <value>{0} is in use.</value>
+  </data>
+  <data name="ErrorOutputFileContainInvalidCharacter" xml:space="preserve">
+    <value>OutputFile setting contains % character.</value>
+  </data>
+  <data name="ErrorUnableLocateAivika" xml:space="preserve">
+    <value>Unable to locate a required Aivika Capture component.</value>
+  </data>
+  <data name="GhostScriptConvertError" xml:space="preserve">
+    <value>Ghostscript conversion error</value>
+  </data>
+  <data name="MinimizeProcessWorkingSetError" xml:space="preserve">
+    <value>Failed to minimize the process working set ({0})</value>
+  </data>
+  <data name="MinimizeProcessWorkingSetMsg" xml:space="preserve">
+    <value>Minimize process working set (current size: {0} bytes)</value>
+  </data>
+  <data name="PrintCapture" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resources\PrintCapture.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+  </data>
+  <data name="ProductCaption" xml:space="preserve">
+    <value>Aivika Printer</value>
```

Comment:
Should perhaps be Aivika Virtual Printer

Reply:
noted.